plot/statetrie_depth.plot

```diff
@@ -0,0 +1,14 @@
+set title "State Trie - Depth"
```

Comment:
I'm not a guru of GNU plots, so tried to generate "pretty" things.
Refer to the mentioned report in the PR description to see the images.